content/en/build/building-with-lotus-APIs.md

```diff
@@ -0,0 +1,646 @@
+
+Developers can also use the API client libraries to utilize the features of Filecoin Network since those libraries take care of the low-level logic of connecting to Lotus node, requesting and handing the response via simple calls, etc. Currently, there are many libraries available:
+
+- [filecoin.js](https://filecoin-shipyard.github.io/filecoin.js/)
+- [filecoin-js-signer](https://github.com/blitslabs/filecoin-js-signer)
+- [Filecoin Signing Tools](https://github.com/Zondax/filecoin-signing-tools)
+
+In this tutorial, we will run a local Lotus node and use the filecoin.js library to walk through the following examples: 
+
+- Setting up a local Lotus node and connecting to [Calibration Testnet](https://docs.filecoin.io/networks/#calibration)
+- Using filecoin.js to write code to interact with Filecoin Network to
+   - query basic blockchain info
+   - manage Lotus wallet
+   - manage light wallet
+   - make a storage deal
+
+## Prerequisites
+
+Before getting started, you should be familiar with:
+
+- [How Filecoin works](https://docs.filecoin.io/about-filecoin/how-filecoin-works/)
+- [How to install a Lotus node for the Calibration network](https://lotu.sh/lotus/install/prerequisites/#supported-platforms)
+- [How to use Lotus command line](https://lotu.sh/lotus/manage/lotus-cli/)
+
+Also, make sure you have all the dependencies installed:
+
+- Fully Synced Calibration TestNet Lotus node
+- node.js
+- npm
+
+## Setup
+
+Before you start writing code to interact with hosted local Lotus node, you should make sure everything is set up properly. Especially ensure you have a local Lotus node fully synced for Calibration TestNet. 
+
+### Set up lotus node
+
+If you have not installed a Lotus node locally and fully synced for Calibration TestNet, you should go back and do it following the guild to [Install and launch a Lotus node](https://lotus.filecoin.io/docs/set-up/install/).
+
+To sum up, you need to:
+
+1. Download Lotus source code for the latest release
+
+   ```shell
+   git clone https://github.com/filecoin-project/lotus.git
+   cd lotus/
+   git checkout <vX.X.X> # tag for a release, currently v1.15.0
+   ```
+
+2. Build and install Lotus for Calibnet
+
+   ```shell
+   make clean calibnet # Calibration with min 32GiB sectors
+   sudo make install
+   ```
+
+3. Fast sync using latest snapshot (still need several hours to fully sync the chain)
+
+   ```shell
+   lotus daemon --import-snapshot <path-to-calibration-snapshot>
+   ```
+
+4. [Create a wallet](https://lotu.sh/lotus/manage/manage-fil/#creating-a-wallet) and request some TestNet FIL from [Faucet](https://faucet.calibration.fildev.network/)
+
+4. [Enable remote API access to your Lotus node](https://lotu.sh/developers/api-access/)
```

Comment:
Again, don't use `lotu.sh` links. Use `docs.lotus.filecoin`

Reply:
I will update all the lotu.sh.
This PR was created before we started using `docs.lotus.filecoin`